factcast-internal-dep-boot/pom.xml

```diff
@@ -10,9 +10,6 @@
   <artifactId>factcast-internal-dep-boot</artifactId>
   <packaging>pom</packaging>
   <properties>
-    <spring-boot.version>2.3.4.RELEASE</spring-boot.version>
```

Comment:
Why aren't these properties not necessary anymore?

Reply:
should not override the inherited value